buildvertical.targets

```diff
@@ -27,13 +27,21 @@
     </MSBuild>
 
     <ItemGroup>
-      <!-- transform back to project -->
-      <ProjectWithConfiguration Include="@(_ProjectBestConfigurations->'%(OriginalItemSpec)')">
-        <AdditionalProperties>Configuration=%(Identity);%(_ProjectBestConfigurations.AdditionalProperties)</AdditionalProperties>
+      <!-- transform back to project keeping the original item order. -->
```

Comment:
Trying to avoid batching which was causing the order of the projects to get shuffled.